### PR TITLE
added migration for bookings, updated booking controller partially

### PR DESCRIPTION
**Pull request by Nick-ODonoghue**
added bookings model and bits in the controller

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.save
+    redirect_to booking_path(@booking)
+  end
+
+  private
+  def booking_params
+    params.require(:booking).permit(:data_from, :date_to)
```
Comment (sulphite) [inline app/controllers/bookings_controller.rb:18]: typo? should be date_from

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class BookingsController < ApplicationController
+  def show
+    @booking = Booking.find(params[:id])
+  end
+
+  def new
+    @booking = Bookings.new
```
Comment (sulphite) [inline app/controllers/bookings_controller.rb:7]: change to Booking singular

**Review by sulphite — CHANGES_REQUESTED** [review]
typos, looks good otherwise

File: app/controllers/bookings_controller.rb (hunk 2 above)
Comment (Nick-ODonoghue) [reply app/controllers/bookings_controller.rb:7]: change made

**Pull request merged**